<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SqlPageControleUI</class>
 <widget class="QWidget" name="SqlPageControleUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,1,0,0,0,0">
   <item>
    <widget class="QPushButton" name="btPre">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>◀</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="currentPage">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="totalPage">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>跳转至</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btGo">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>GO</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btNext">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>25</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Algerian</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>▶</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
